src/main/features/response/views/statement-of-means/residence.njk

```diff
@@ -0,0 +1,55 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioOption, saveAndContinueButton, textInput %}
+
+{% set title = 'Money claim' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Where are you living?') }}</h1>
+
+      <form
+        method="POST"
+        class="analytics-submit-event-trigger"
+        data-event-action="Response: Statement of means: Select defendant response option"
+        data-event-label-from="type"
+        novalidate>
+
+        {{ csrfProtection(csrf) }}
+
+        {% set error = form.errorFor('type') %}
+        <div class="form-group {% if error %} form-group-error {% endif %}">
+          <fieldset id="type[label]">
+            <legend class="visually-hidden">{{ t('Where are you living?') }}</legend>
+            {% if error %}<span class="error-message">{{ t(error) }}</span>{% endif %}
+
+            {% for type in ResidenceType.all() %}
```

Comment:
I would suggest implementing `ResidenceType#except` method as we tend to do. Then we could do `ResidenceType.except(ResidenceType.OTHER)` in the view. Alternatively we could do that in the route file.